addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -104,6 +105,16 @@ var SearchableMediaWidget = MediaWidget.extend({
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+    _onSearchKeydown: function (ev) {
+        // If the template contains a form that has only one input, the enter
+        // will reload the page as the html 2 specify this behavior.
```

Comment:
I'd write "2.0" instead of "2" because otherwise one might not understand what you mean.